tests/cypress/tests/02-search/02-pod-logs.spec.js

```diff
@@ -23,7 +23,7 @@ const clusterModes = [
   {
     label: 'Managed',
     valueFn: () => cliHelper.getTargetManagedCluster(),
-    skip: Cypress.env('SKIP_MANAGED_CLUSTER_TEST'),
+    skip: true, // Disabling the managed cluster tests until the console squad patches the issue for the logs action.
```

Comment:
Can you output a message to the test output to get a reminder that this is disabled?